src/electron/device-connect-view/components/window-title.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { css } from '@uifabric/utilities';
+import * as React from 'react';
+import { NamedSFC } from '../../../common/react/named-sfc';
+
+export interface WindowTitleProps {
+    title: string;
+    children?: JSX.Element;
+}
+
+export const WindowTitle = NamedSFC<WindowTitleProps>('WindowTitle', (props: WindowTitleProps) => {
+    return (
+        <header className="ms-Grid window-title">
+            <div className={css('ms-Grid-row')}>
+                {props.children}
+                <div role="heading" aria-level={1} className="ms-Grid-col header-text">
```

Comment:
Why a `<div role="heading" aria-level={1}>` rather than just an `<h1>`?

Reply:
No particular reason--modeled this after [header.tsx](https://github.com/microsoft/accessibility-insights-web/blob/master/src/popup/components/header.tsx)

Reply:
Now an h1